### PR TITLE
UNGEGN poster sample page

### PR DESCRIPTION
**Pull request by webdev778**
Implement https://github.com/interscript/interscript/issues/438

image

### CODE REVIEW

File: src/containers/samples.json [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+    "isoName": "national: Kunrei-siki/Mod. Hepburn",
+    "sysCode": "mext-jpn-Hrkt-Latn-1954",
+    "samples": ["日本","東京"],
+    "result": []
+  },
+  {
+    "lang": "Kannada",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "bis-knd-Knda-Latn-13194-1991",
+    "samples": [],
```
Comment (ronaldtse) [inline src/containers/samples.json:118]: Missing sample: 
Screen Shot 2020-09-29 at 9 07 50 AM

File: src/containers/samples.json [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+    "result": []
+  },
+  {
+    "lang": "Kazakh",
+    "isoName": "BGN/PCGN 1979",
+    "sysCode": "odni-kaz-Cyrl-Latn-2015",
```
Comment (ronaldtse) [inline src/containers/samples.json:124]: Should be bgnpcgn system

File: src/containers/samples.json [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+    "samples": ["Қазақстан","Астана"],
+    "result": []
+  },
+  {
+    "lang": "Khmer",
+    "isoName": "UN 1972, II/10",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:131]: We're missing a un-khm-Khmr-Latn-1972 system.

File: src/containers/samples.json [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+    "samples": ["កមពុជ","ភនំេពញ"],
+    "result": []
+  },
+  {
+    "lang": "Kirghiz",
+    "isoName": "BGN/PCGN 1979",
+    "sysCode": "odni-kir-Cyrl-Latn-2015",
```
Comment (ronaldtse) [inline src/containers/samples.json:138]: Should be bgnpcgn system

File: src/containers/samples.json [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Korean",
+    "isoName": "ISO/TR 11941:1996",
+    "sysCode": "iso-kor-Hang-Latn-1996-method1",
+    "samples": ["평양","서울"],
+    "result": []
+  },
+  {
+    "lang": "Korean",
+    "isoName": "ISO/TR 11941:1996",
+    "sysCode": "iso-kor-Hang-Latn-1996-method1",
```
Comment (ronaldtse) [inline src/containers/samples.json:152]: `method2`?

File: src/containers/samples.json [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
+    "samples": ["평양","서울"],
+    "result": []
+  },
+  {
+    "lang": "Lao",
+    "isoName": "national 1966",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:159]: We're missing the Lao system.

File: src/containers/samples.json [hunk 7]
```diff
@@ -0,0 +1,303 @@
...
+    "samples": ["ລາວ","ວຽງຈັນ"],
+    "result": []
+  },
+  {
+    "lang": "Macedonian Cyrillic",
+    "isoName": "UN 1977, III/11",
+    "sysCode": "odni-mkd-Cyrl-Latn-2015",
```
Comment (ronaldtse) [inline src/containers/samples.json:166]: Should be a UN system.

File: src/containers/samples.json [hunk 8]
```diff
@@ -0,0 +1,303 @@
...
+    "samples": ["Скопје","Битола"],
+    "result": []
+  },
+  {
+    "lang": "Malayalam",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:173]: Missing the Malayalam system.

File: src/containers/samples.json [hunk 9]
```diff
@@ -0,0 +1,303 @@
...
+    "samples": [],
+    "result": []
+  },
+  {
+    "lang": "Maldivian",
+    "isoName": "national 1987",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:180]: Missing Maldivian system.

File: src/containers/samples.json [hunk 10]
```diff
@@ -0,0 +1,303 @@
...
+    "isoName": "BGN/PCGN 1964",
+    "sysCode": "bgnpcgn-mon-Cyrl-Latn-1964",
+    "samples": ["Монгол улс","Улаанбаатар"],
+    "result": []
+  },
+  {
+    "lang": "Nepalese",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "un-nep-Deva-Latn-1972",
+    "samples": [],
```
Comment (ronaldtse) [inline src/containers/samples.json:202]: Missing samples.

File: src/containers/samples.json [hunk 11]
```diff
@@ -0,0 +1,303 @@
...
+    "result": []
+  },
+  {
+    "lang": "Oriya",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "bis-ori-Orya-Latn-13194-1991",
+    "samples": [],
```
Comment (ronaldtse) [inline src/containers/samples.json:209]: Missing samples.

File: src/containers/samples.json [hunk 12]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Pashto",
+    "isoName": "BGN/PCGN 1968",
+    "sysCode": "bgnpcgn-prs-Arab-Latn-2007",
+    "samples": ["افغانستان","کابل"],
+    "result": []
+  },
+  {
+    "lang": "Persian",
+    "isoName": "UN 1967, I/13",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:222]: Missing UN Persian system.

File: src/containers/samples.json [hunk 13]
```diff
@@ -0,0 +1,303 @@
...
+    "samples": ["تھران","ايران"],
+    "result": []
+  },
+  {
+    "lang": "Punjabi",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:229]: Missing Punjabi system.

File: src/containers/samples.json [hunk 14]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Russian",
+    "isoName": "UN 1987, V/18",
+    "sysCode": "",
+    "samples": ["Россия","Москва"],
+    "result": []
+  },
+  {
+    "lang": "Serbian",
+    "isoName": "UN 1977, III/11; 2002, VIII/14",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:243]: Missing UN Serbian.

File: src/containers/samples.json [hunk 15]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Punjabi",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "",
+    "samples": ["ਪੰਜਾਬ","ਚੰ ਡੀਗੜ"],
+    "result": []
+  },
+  {
+    "lang": "Russian",
+    "isoName": "UN 1987, V/18",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:236]: Missing UN Russian.

File: src/containers/samples.json [hunk 16]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Sinhalese",
+    "isoName": "provisional 1972",
+    "sysCode": "",
+    "samples": [],
+    "result": []
+  },
+  {
+    "lang": "Tajik",
+    "isoName": "BGN/PCGN 1994",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:257]: Missing BGNPCGN Tajik

File: src/containers/samples.json [hunk 17]
```diff
@@ -0,0 +1,303 @@
...
+    "isoName": "UN 1977, III/11; 2002, VIII/14",
+    "sysCode": "",
+    "samples": ["Србија","Београд"],
+    "result": []
+  },
+  {
+    "lang": "Sinhalese",
+    "isoName": "provisional 1972",
+    "sysCode": "",
+    "samples": [],
```
Comment (ronaldtse) [inline src/containers/samples.json:251]: Missing Sinhalese system and samples.

File: src/containers/samples.json [hunk 18]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Tajik",
+    "isoName": "BGN/PCGN 1994",
+    "sysCode": "",
+    "samples": ["Тоҷикистон","Душанбе"],
+    "result": []
+  },
+  {
+    "lang": "Tamil",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:264]: Missing UN Tamil system.

File: src/containers/samples.json [hunk 19]
```diff
@@ -0,0 +1,303 @@
...
+    "samples": ["தமி நா ","இலைக"],
+    "result": []
+  },
+  {
+    "lang": "Telugu",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:271]: Missing Telugu system and samples.

File: src/containers/samples.json [hunk 20]
```diff
@@ -0,0 +1,303 @@
...
+    "samples": [],
+    "result": []
+  },
+  {
+    "lang": "Thai",
+    "isoName": "UN 1967, I/14",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:278]: Missing UN Thai system.

File: src/containers/samples.json [hunk 21]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Tigrinya",
+    "isoName": "BGN/PCGN 2007",
+    "sysCode": "",
+    "samples": ["ኤርትራ","ኣስመራ"],
+    "result": []
+  },
+  {
+    "lang": "Ukrainian",
+    "isoName": "national 2007",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:292]: Missing Ukrainian national system

File: src/containers/samples.json [hunk 22]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Thai",
+    "isoName": "UN 1967, I/14",
+    "sysCode": "",
+    "samples": ["ไทย","กรุงเทพฯ"],
+    "result": []
+  },
+  {
+    "lang": "Tigrinya",
+    "isoName": "BGN/PCGN 2007",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:285]: Missing Tigrinya system.

File: src/containers/samples.json [hunk 23]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Ukrainian",
+    "isoName": "national 2007",
+    "sysCode": "",
+    "samples": ["Україна"],
+    "result": ["Україна","Київ"]
+  },
+  {
+    "lang": "Urdu",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:299]: Missing UN Urdu system.

File: src/containers/samples.json [hunk 24]
```diff
@@ -0,0 +1,303 @@
...
+    "isoName": "UN 1977, III/13; 2007, IX/9",
+    "sysCode": "icao-heb-Hebr-Latn-9303",
+    "samples": ["מְ דִ ינַת י ִׂשְ ָראֵ ל"],
+    "result": []
+  },
+  {
+    "lang": "Hindi",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "un-hin-Deva-Latn-2016",
+    "samples": [],
```
Comment (ronaldtse) [inline src/containers/samples.json:104]: Missing samples.

File: src/containers/samples.json [hunk 25]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Greek",
+    "isoName": "UN 1987, V/19",
+    "sysCode": "un-ell-Grek-Latn-1987-tl",
+    "samples": ["Ελλάδα","Αθήνα"],
+    "result": []
+  },
+  {
+    "lang": "Gujarati",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:89]: Missing UN Gujarati system.

File: src/containers/samples.json [hunk 26]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Chinese",
+    "isoName": "UN 1977, III/8",
+    "sysCode": "acadsin-zho-Hani-Latn-2002",
+    "samples": ["中国","北京"],
+    "result": []
+  },
+  {
+    "lang": "Dzongkha",
+    "isoName": "national 1997",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:68]: Missing Dzongkha system.

File: src/containers/samples.json [hunk 27]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Bulgarian",
+    "isoName": "UN 1977, III/10",
+    "sysCode": "",
+    "samples": ["Република България"],
+    "result": []
+  },
+  {
+    "lang": "Burmese",
+    "isoName": "BGN/PCGN 1970",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:47]: Missing BGNPCGN Burmese system

File: src/containers/samples.json [hunk 28]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Bengali",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "un-ben-Beng-Latn-2016",
+    "samples": ["",""],
+    "result": []
+  },
+  {
+    "lang": "Bulgarian",
+    "isoName": "UN 1977, III/10",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:40]: Missing UN Bulgarian system

File: src/containers/samples.json [hunk 29]
```diff
@@ -0,0 +1,303 @@
...
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "",
+    "samples": [],
+    "result": []
+  },
+  {
+    "lang": "Bengali",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "un-ben-Beng-Latn-2016",
+    "samples": ["",""],
```
Comment (ronaldtse) [inline src/containers/samples.json:34]: Missing samples for Bengali

File: src/containers/samples.json [hunk 30]
```diff
@@ -0,0 +1,303 @@
...
+    "lang": "Armenian",
+    "isoName": "BGN/PCGN 1981",
+    "sysCode": "bgnpcgn-arm-Armn-Latn-1981",
+    "samples": ["Հայաստան","Երևան"],
+    "result": []
+  },
+  {
+    "lang": "Assamese",
+    "isoName": "UN 1972, II/11; 1977, III/12",
+    "sysCode": "",
```
Comment (ronaldtse) [inline src/containers/samples.json:26]: Missing UN Assamese system and samples.

File: src/containers/samples.json [hunk 31]
```diff
@@ -0,0 +1,303 @@
...
+    "isoName": "UN 1967, I/17",
+    "sysCode": "un-amh-Ethi-Latn-2016",
+    "samples": ["ኢትዮጵያ", "አዲስ አበባ"],
+    "result": []
+  },
+  {
+    "lang": "Arabic",
+    "isoName": "UN 1972, II/8",
+    "sysCode": "un-ara-Arab-Latn-1972",
+    "samples": [],
```
Comment (ronaldtse) [inline src/containers/samples.json:13]: Missing samples for Arabic.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

**Comment by ronaldtse** [comment]
1. Can you file those missing systems to be implemented? (or find the correct systems)
2. For the non-implemented systems, we can still show the samples, but in results show "(to be implemented)"

File: src/containers/samples.json (hunk 5 above)
Comment (webdev778) [reply src/containers/samples.json:152]: method1 is for North Korea system and method2 is for South Korea. I'm wondering which one should it be used for transliteration of examples. I used methoed1

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: src/containers/samples.json (hunk 5 above)
Comment (ronaldtse) [reply src/containers/samples.json:152]: Pyongyang should use method1 and Seoul should use method2. Can we do that?

**Comment by webdev778** [comment]
Currently, missing systems are as following

Total : 23
-Assamese [UN 1972, II/11; 1977, III/12]
-Bulgarian [UN 1977, III/10]
-Burmese [BGN/PCGN 1970]
-Dzongkha [national 1997]
-Gujarati [UN 1972, II/11; 1977, III/12]
-Kazakh [BGN/PCGN 1979]
-Khmer [UN 1972, II/10]
-Kirghiz [BGN/PCGN 1979]
-Lao [national 1966]
-Macedonian Cyrillic [UN 1977, III/11]
-Malayalam [UN 1972, II/11; 1977, III/12]
-Maldivian [national 1987]
-Persian [UN 1967, I/13]
-Punjabi [UN 1972, II/11; 1977, III/12]
-Serbian [UN 1977, III/11; 2002, VIII/14]
-Sinhalese [provisional 1972]
-Tajik [BGN/PCGN 1994]
-Tamil [UN 1972, II/11; 1977, III/12]
-Telugu [UN 1972, II/11; 1977, III/12]
-Thai [UN 1967, I/14] 
-Tigrinya [BGN/PCGN 2007]
-Ukrainian [national 2007]
-Urdu [UN 1972, II/11; 1977, III/12]

File: src/containers/samples.json [hunk 32]
```diff
@@ -0,0 +1,303 @@
...
+    "result": []
+  },
+  {
+    "lang": "Armenian",
+    "isoName": "BGN/PCGN 1981",
```
[the remark this answers is not in the document]
Comment (webdev778) [reply src/containers/samples.json:18]: you mean sysCode ?

[the remark this answers is not in the document]
Comment (ronaldtse) [reply src/containers/samples.json:18]: I meant what I wrote, "BGN/PCGN 1981" is the system name. `bgnpcgn-arm-xxx` is the ISO code ([ISO 24229](https://calconnect.gitlab.io/tc-localization/csd-transcription-systems/documents/iso-24229.html))

**Comment by webdev778** [comment]
Add missing samples and fix some system names

**Comment by ronaldtse** [comment]
@webdev778 https://github.com/interscript/interscript/pull/297 already implements the UN Ukrainian system `un-ukr-cyrl-latn-1998`. Can you double check?

**Pull request merged**